test/mpi/include/mtest_common.h

```diff
@@ -16,6 +16,7 @@ typedef enum {
     MTEST_MEM_TYPE__UNREGISTERED_HOST,
     MTEST_MEM_TYPE__REGISTERED_HOST,
     MTEST_MEM_TYPE__DEVICE,
+    MTEST_MEM_TYPE__SHARED,
```

Comment:
Is ze "shared" memory similar to cuda "managed" memory?

Reply:
Yes. they are similar.